README.md

```diff
@@ -52,3 +52,32 @@ See [pipelines/README.md](pipelines/README.md)
     * `oc edit cm cluster-monitoring-config`
     * Set variable `enableUserWorkload` to `true`
   * Edit contents of [thanos-secret](acm/odh-core/acm-observability/secrets/thanos.yaml) file.
+
+## Development
+
+### Deploying a model service using the ACM GitOps flow
+
+Two different model applications were deployed above in the "Infrastructure Configuration" section.
+You can also deploy your own isolated version of either of these to test changes to the configurations, with the following steps:
+
+1. Push your changes to a branch of your clone of this repo. Ensure that you change `namespace` in either `test/gitops/bike-rental-app/kustomization.yaml` or `test/gitops/tensorflow-housing-app/kustomization.yaml`, or both, as appropriate.
+2. Substitute the values of the `GIT_REPO_URL`, `GIT_BRANCH`, and `TEST_NAMESPACE` variables in this `make` command appropriately:
+NOTE: Escape the `:` in the `https://` protocol part of the `GIT_REPO_URL` value
+```bash
+make -e GIT_REPO_URL="https\://github.com/opendatahub-io/ai-edge" \
+     -e GIT_BRANCH=my-git-branch \
+     -e TEST_NAMESPACE=my-test-namespace \
+     test-acm-bike-rental-app-deploy # or test-acm-tensorflow-housing-deploy
+```
+3. Create a `ManagedClusterSetBinding binding for the namespace, by running the following (modify `metadata.namespace` appropriately):
```

Comment:
There's a bunch of kustomize tetris needed to be able to include the `ManagedClusterSetBinding` resource by default. I'll attempt that in a separate PR, since it affects the production versions of these apps too, not just this development stuff.